src/promptflow-tracing/promptflow/tracing/_trace.py

```diff
@@ -143,6 +143,8 @@ def collect_openai_tokens_for_parent_span(self, span):
                     merged_tokens = {
                         key: self._span_id_to_tokens[parent_span_id].get(key, 0) + tokens.get(key, 0)
                         for key in set(self._span_id_to_tokens[parent_span_id]) | set(tokens)
+                        if isinstance(self._span_id_to_tokens[parent_span_id].get(key, 0), int)
```

Comment:
Thank you for the fix. if the value is not `int`, should we count it as 0?

Reply:
Discussed offline. Please add a comment to state why tokens values are not `int` sometime.